docs/LocalHosting.md

```diff
@@ -41,9 +44,6 @@ npm run install-all
 
 # Start Application Locally
 
-Ensure that MySQL Server service has started.
-(How to check? Click [here](LocalMySqlSetup.md#start-mysql-service).)
-
 Start all services using the following commands:
```

Comment:
Perhaps there a need to specify to 'ensure that xxx service has started' for RabbitMQ/Redis/Ubuntu?